styleguide/source/_patterns/02-molecules/inline-bio.json

```diff
@@ -11,11 +11,13 @@
       "text": "Steven J. Stack, MD",
       "class": "ama__inline-bio__title ama__link--blue"
     },
-    "paragraphs": [{
+    "metaHeading": [{
+      "level": "3",
```

Comment:
This is using headings out of order which is against accessibility best practices (https://webaim.org/techniques/semanticstructure/). Instead of changing the page semantics we should be making these elements look like an `h3` by using `@extend` on the class that is already being used on these elements.